app/views/interrupt/DDInterruptNoDD.scala.html

```diff
@@ -15,7 +15,7 @@
  *@
 
 @this(mainTemplate: MainTemplate,
-        govukDetails: GovukDetails,
+        govukInsetText : GovukInsetText,
```

Comment:
This isn't something you have changed but I've just seen that the `Back` link has no Welsh translation. I believe this is because it does not have its `content` parameter set, so it defaults  to "Back" regardless of language. Could you correct this while you're here?